src/io/IOHelpers.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "molecules/Component.h"
+#include "utils/Random.h"
+
+class ParticleContainer;
+class DomainDecompBase;
+
+namespace IOHelpers {
+
+/**
+ * Removes the momentum from the simulation.
+ * Will first sum up the overall momentum of the simulation and then apply a shift to all particles, s.t., there is no
+ * more total momentum.
+ * @param particleContainer
+ * @param components
+ * @param domainDecomp
+ */
+void removeMomentum(ParticleContainer* particleContainer, const std::vector<Component>& components,
+					DomainDecompBase* domainDecomp);
+
+/**
+ * Assigns random velocities based on the given temperature to all particles in the given particleContainer.
+ *
+ * @param particleContainer
+ * @param domainDecomp
+ * @param temperature
+ */
+void initializeVelocityAccordingToTemperature(ParticleContainer* particleContainer, DomainDecompBase* domainDecomp,
+											  double temperature);
+
+/**
+ * Makes the particle ids of a container unique (throughout all ranks).
```

Comment:
```suggestion
 * Makes the particle ids of a container unique (throughout all ranks).
 * First an exclusive scan is used to get the number of particles of all processes with lower rank.
 * Each rank then adds that number to the ids of its particles.
```

Reply:
Better. The issue I still have with the function is that its name describes what it is used for but not what it does. However, at least since this documentation now does this it's ok and I would accept it.